### DISCUSSION TITLE
Additional Headers Doesn't work!

### DISCUSSION THREAD

**Discussion by remchuk** [netbox-community/netbox#12869] [Help Wanted!] [answered]
Hey people!

I'm trying to make an additional header in a webhook yet when I print on the listener the POST request using : json.loads(json.dumps(<POST-REQ>)) 
I have no additional headers.

For example I have a header : Token : abc

I use NetBox 3.3.4 , anyone know why is that?

**Comment by kkthxbye-code** [comment]
> when I print on the listener the POST request using : json.loads(json.dumps())

This makes no sense. Show your code instead.

**Reply by remchuk** [reply]
```
def tester():
    testy(request.get_json())

def testy(post_req_json):
    post_req_json = json.loads(json.dumps(post_req_json)
    print(post_req_json)
```

**Reply by kkthxbye-code** [reply, marked as answer]
If request is your response object, headers will be in `request.headers`.

**Reply by remchuk** [reply]
Thank you!

Do you know perhaps how can I add it to the data section ?

**Reply by kkthxbye-code** [reply]
You would have to create your own body template. You can't add to the body, so you will have to create it from scratch. Shouldn't be to bad, I think you can use the `tojson` filter.
